### ISSUE TITLE
SSL Certificate Error / Unsafe Network

### ISSUE THREAD

**Issue by itsadamslife** [widewing/ha-toyota-na#21]
I started getting errors the last couple of days in Home Assistant that my integration can't connect due to an invalid SSL certificate. Initially, I thought it had something to do with my pfSense router as I recently moved my HA to a different VLAN but I found nothing in the logs to indicate a problem. I noticed today that the Toyota app on my Android phone now pops up saying that I'm trying to connect from an unsafe network when connecting from my wifi. The app works fine when wifi is turned off. Connecting to Toyota through a VPN on my network works fine. 

The following is from my log file:

```
This error originated from a custom integration.

Logger: custom_components.toyota_na
Source: custom_components/toyota_na/__init__.py:98
Integration: Toyota (North America) (documentation, issues)
First occurred: 09:59:37 (2 occurrences)
Last logged: 09:59:42

Error fetching data
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 986, in _wrap_create_connection
    return await self._loop.create_connection(*args, **kwargs)  # type: ignore[return-value]  # noqa
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1081, in create_connection
    transport, protocol = await self._create_connection_transport(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1111, in _create_connection_transport
    await waiter
  File "/usr/local/lib/python3.9/asyncio/sslproto.py", line 528, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/local/lib/python3.9/asyncio/sslproto.py", line 188, in feed_ssldata
    self._sslobj.do_handshake()
  File "/usr/local/lib/python3.9/ssl.py", line 944, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1129)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/config/custom_components/toyota_na/__init__.py", line 98, in update_vehicles_status
    vehicles = await client.get_user_vehicle_list()
  File "/usr/local/lib/python3.9/site-packages/toyota_na/client.py", line 44, in get_user_vehicle_list
    return await self.api_get("v2/vehicle/guid")
  File "/usr/local/lib/python3.9/site-packages/toyota_na/client.py", line 38, in api_get
    return await self.api_request("GET", endpoint, header_params)
  File "/usr/local/lib/python3.9/site-packages/toyota_na/client.py", line 27, in api_request
    async with session.request(method, urljoin(self.API_GATEWAY, endpoint), headers=headers, **kwargs) as resp:
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client.py", line 1138, in __aenter__
    self._resp = await self._coro
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client.py", line 535, in _request
    conn = await self._connector.connect(
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 542, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 907, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 1206, in _create_direct_connection
    raise last_exc
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 1175, in _create_direct_connection
    transp, proto = await self._wrap_create_connection(
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 988, in _wrap_create_connection
    raise ClientConnectorCertificateError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorCertificateError: Cannot connect to host oneapi.telematicsct.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1129)')]
```

Screenshots from the same error and unsafe network error in the Toyota app:
https://ibb.co/jr5WPVC
https://ibb.co/1rzZY7d

**Comment by widewing** [comment]
What about open https://http://oneapi.telematicsct.com/ in browser?

**Comment by itsadamslife** [comment]
```
This site can’t be reached
The webpage at https://oneapi.telematicsct.com/ might be temporarily down or it may have moved permanently to a new web address.
ERR_SSL_KEY_USAGE_INCOMPATIBLE
```

I get the JSON readout that says unauthorized when opening via VPN. 

When connected through VPN and I ping oneapi.telematicsct.com, it resolves to 157.245.183.96 but when I ping without VPN, it resolves to 172.16.222.1. Now I'm thinking it might be my network but I can't find anything and I'm not having any issues elsewhere that I've noticed.

**Comment by itsadamslife** [comment]
I found the issue...
I have pfBlocker on my router, which for some reason started blocking *.telematicsct.com. 
All is well now! Thank you for responding!
